src/components/event-list/index.js

```diff
@@ -25,6 +26,17 @@ class EventList extends Component {
         <List>
           {events
             .filter(e => e.name.toLowerCase().includes(query.toLowerCase()))
+            .map(e => {
+              e.date = new Date(e.date)
```

Comment:
Just curious, what's the purpose of creating a new date? Is there some sort of scoping issue?

Reply:
`new Date` parses a date from a string